v22.2/ui-statements-page.md

```diff
@@ -7,18 +7,65 @@ docs_area: reference.db_console
 
 {% include {{ page.version.version }}/ui/admin-access.md %}
 
-The **Statements** page  of the DB Console helps you:
+The **Statements** page offers two views on statements: **Statement Fingerprints**, which represents one or more completed SQL statements and **Active Executions**,
+which represents individual statement executions in progress.
+
+You choose a view by selecting the **Statement Fingerprints** or **Active Executions** radio button. The selection is retained when you switch between the Statements and Transactions tabs on the SQL Activity page.
+
+{{site.data.alerts.callout_success}}
+If you haven't yet executed any queries in the cluster as a user, this page will be blank.
+{{site.data.alerts.end}}
+
+## Statement Fingerprints view
+
+The **Statements Fingerprints** view helps you:
 
 - Identify frequently executed or high latency [SQL statements](sql-statements.html).
 - View SQL statement fingerprint [details](#statement-fingerprint-page).
 - Download SQL statement [diagnostics](#diagnostics) for troubleshooting.
 
-To view this page, click **SQL Activity** in the left-hand navigation of the DB Console. The **Statements** tab is selected.
+{% if page.cloud != true %}
+To view this page, click **SQL Activity** in the left-hand navigation of the DB Console.
+{% else %}
+To view this page, click **SQL Activity** in the left-hand navigation of the {{ site.data.products.db }} Console.
+{% endif %}
 
-{{site.data.alerts.callout_success}}
-If you haven't yet executed any queries in the cluster as a user, this page will be blank.
+The **Statements** tab is selected. The **Statement Fingerprints** radio button is selected and the [Statements table](#statements-table) displays.
```

Comment:
Is this right?

```suggestion
When the **Statements** tab is selected, click the **Statement Fingerprints** radio button. The [Statements table](#statements-table) displays.
```

Reply:
No. When you click SQL Activity, the Statements tab and Statement Fingerprints button are selected by default.

Reply:
Ah, I understand now. In that case, these look well worded.